### DISCUSSION TITLE
Dialog/Drawer: How to set the size/width of a dialog or drawer?

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/ark#2951] [Q&A] [answered]
Figured out you can set the width by setting it to the Positioner but I am not sure if that is the correct place honestly. Used to ChakraUI, the Size of Drawer or Dialogs was set on the Root component instead.

**Comment by segunadebayo** [comment, marked as answer]
In Ark UI, the actual width should be set on dialog content. Chakra uses a theming system that’s why it appears you set the size on the root element.

**Reply by xeinebiu** [reply]
> In Ark UI, the actual width should be set on dialog content. Chakra uses a theming system that’s why it appears you set the size on the root element.

When I set on the content, nothing happens. Using ParkUI to test and it only works when I set the size/width on the Positioner.
